Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:8.11.4
+
+WORKDIR /app/website
+
+EXPOSE 3000 35729
+COPY ./docs /app/docs
+COPY ./website /app/website
+RUN yarn install
```

Comment:
Does docusaurus require `yarn` or can we use npm?

Reply:
That's autogenerated